src/SFA.DAS.AANHub.Application/Partners/Queries/GetPartnerMemberResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Partners.Queries
+{
+    public class GetPartnerMemberResult
+    {
+        public Guid MemberId { get; set; }
+        public string? Email { get; set; }
+        public string? Organisation { get; set; }
+
+        public static implicit operator GetPartnerMemberResult?(Partner partner)
```

Comment:
Partner should be nullable rather than `GetPartnerMemberResult`

Reply:
Afraid not as null is passed to GetPartnerMemberResult (line 14). The code is very similar to the [GETAdmin endpoint](https://github.com/SkillsFundingAgency/das-aan-hub-api/pull/44/files#diff-2ce896ea44c5aadf66bc1f10033e2f51f33acf97af157a7782467aafe282f89a)